omod/src/main/java/org/openmrs/module/appointments/web/mapper/AppointmentMapper.java

```diff
@@ -272,9 +274,24 @@ private Map createPatientMap(Patient p) {
         map.put("name", p.getPersonName().getFullName());
         map.put("uuid", p.getUuid());
         map.put("identifier", p.getPatientIdentifier().getIdentifier());
-        map.put("age", p.getAge());
-        map.put("gender", p.getGender());
         map.putAll(p.getActiveIdentifiers().stream().filter(e -> e.getIdentifierType() != null).collect(Collectors.toMap(e -> e.getIdentifierType().toString().replaceAll("[- ]", ""), e -> e.getIdentifier())));
+        for (PatientIdentifier patientIdentifier : p.getIdentifiers()) {
```

Comment:
This doesn't look right/appropriate to me.  Too much hard-coding.  Why don't we just use standard OpenMRS REST responses and conversions for this data?